Extensions/super_markdown_editor.js

```diff
@@ -0,0 +1,262 @@
+//* TITLE Super Markdown Editor **//
+//* VERSION 1.0 REV A **//
+//* DESCRIPTION Add live previews and other functionality to the Markdown post editor **//
+//* DEVELOPER new-xkit **//
+//* FRAME false **//
+//* BETA false **//
```

Comment:
I think this should be `BETA true` as this would be the extension's first release